storm-core/src/jvm/org/apache/storm/task/ShellBolt.java

```diff
@@ -158,8 +158,8 @@ public void execute(Tuple input) {
 
             _pendingWrites.putBoltMsg(boltMsg);
         } catch(InterruptedException e) {
-            String processInfo = _process.getProcessInfoString() + _process.getProcessTerminationInfoString();
-            throw new RuntimeException("Error during multilang processing " + processInfo, e);
+            // It's likely that Bolt is shutting down so no need to throw RuntimeException
```

Comment:
@abhishekagarwal87 Do you have update on this? If your work is expected to take longer, please review this first and we could handle your improvement by separate issue.